src/test/mathutiltest.cpp

```diff
@@ -64,19 +64,26 @@ TEST_F(MathUtilTest, Denormal) {
     _MM_SET_DENORMALS_ZERO_MODE(_MM_DENORMALS_ZERO_OFF);
     _MM_SET_FLUSH_ZERO_MODE(_MM_FLUSH_ZERO_OFF);
 
-    volatile float fDenormal = std::numeric_limits<float>::min() / 2.0f;
+    // Note: The volatile keyword makes sure that the devision is executed on the target
+    // and not by the pre-processor. In case of clang >= 15 the pre-processor flushes to
+    // zero with -ffast-math enabled.
```

Comment:
```suggestion
    // Note: The volatile keyword makes sure that the division is executed on the target
    // and not by the pre-processor. In case of clang >= 15 the pre-processor flushes to
    // zero with -ffast-math enabled.
```